src/parser/priest/shadow/modules/spells/ShadowWordPain.tsx

```diff
@@ -64,26 +49,6 @@ class ShadowWordPain extends Analyzer {
     return spell.damageEffective + spell.damageAbsorbed;
   }
 
-  onCast(event: CastEvent) {
-    this.castedShadowWordPains += 1;
-  }
-
-  onDebuffApplied(event: ApplyDebuffEvent) {
-    this.appliedShadowWordPains += 1;
-
-    if (this.lastCastTimestamp !== 0 && event.timestamp < this.lastCastTimestamp + MS_BUFFER) {
-      this.darkVoidShadowWordPainApplications += 1;
-    }
-  }
-
-  onDebuffRefreshed(event: RefreshDebuffEvent) {
-    this.refreshedShadowWordPains += 1;
-
-    if (this.lastCastTimestamp !== 0 && event.timestamp < this.lastCastTimestamp + MS_BUFFER) {
-      this.darkVoidShadowWordPainRefreshes += 1;
-    }
-  }
-
```

Comment:
Dark void was removed and replaced with searing nightmare. At the moment this isn't needed, will likely be adding back similar functionality for searing nightmare but not at this moment.